deployments/eirini.go

```diff
@@ -65,83 +65,88 @@ func (k Eirini) Describe() string {
 func (k Eirini) Delete(c *kubernetes.Cluster, ui *ui.UI) error {
 	ui.Note().KeeplineUnder(1).Msg("Removing Eirini...")
 
-	releaseDir, err := k.ExtractRelease()
+	owned, err := c.NamespaceOwned(eiriniCoreNamespace)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "failed to check if namespace '%s' is owned or not", eiriniCoreNamespace)
 	}
-	defer func() {
-		err := os.RemoveAll(releaseDir)
+	if owned {
```

Comment:
To make the code a bit more readable and avoid the additional identation, you can move the `else` part on top and return early. Something like:

```
if !c.NamespaceOwned(eiriniCoreNamespace)  {
  ui.Exclamation().Msg("Skipping Eirini because it is not owned by carrier")
  return nil
}

```

Reply:
The same in the other deployments.

Reply:
Done.